### ISSUE TITLE
Get cached toggles when device is offline

### ISSUE THREAD

**Issue by afurtak** [Unleash/unleash-android-proxy-sdk#8]
Scenario:
1. set toggle value to true
2. run the app and get toggle value
3. turn off WiFi and mobile data 
4. restart the app

What I would expect:
Toggle's value should be still equal to true until the app receives new value

Actual behavior:
Toggle's value is equal to false - default value

**Comment by ivarconr** [comment]
Thanks for reporting this. I am a bit puzzled that it is currently not working, why does not the local cache take care of it?

**Comment by afurtak** [comment]
A temporary directory instance is passed as a parameter to Cache constructor. May it be a problem?
```
OkHttpClient.Builder()
        .readTimeout(unleashConfig.httpClientReadTimeout)
        .connectTimeout(unleashConfig.httpClientConnectionTimeout)
        .cache(
            Cache(
                directory = Files.createTempDirectory("unleash_toggles").toFile(),
                maxSize = unleashConfig.httpClientCacheSize
            )
        ).build()
```

**Comment by chriswk** [comment]
Indeed, that will probably be an issue, because the temp directory will be per startup. We'll look into how to make the tempdir persistent across sessions.

**Comment by hadalin** [comment]
Any change on revisiting this? One way would be to have `PersistentToggleCache` besides `InMemoryToggleCache`.
